lib/fog/kubevirt.rb

```diff
@@ -1 +1,13 @@
-require 'fog/kubevirt/compute'
+require "fog/core"
+
+module Fog
+  module Compute
+    autoload :Kubevirt, File.expand_path('../kubevirt/compute', __FILE__)
```

Comment:
@plribeiro3000 this was asked by you via an email.
Could you review ?

Reply:
Looking good to me so far. =)